src/keycloak/connection.py

```diff
@@ -65,11 +69,16 @@ def __init__(self, base_url, headers={}, timeout=60, verify=True, proxies=None):
         :type verify: Union[bool,str]
         :param proxies: The proxies servers requests is sent by.
         :type proxies: dict
+        :param cert: An SSL certificate used by the requested host to authenticate the client.
+            Either a path to an SSL certificate file, or two-tuple of
+            (certificate file, key file).
+        :type cert: Union[str,Tuple[str,str]]
         """
         self.base_url = base_url
         self.headers = headers
         self.timeout = timeout
         self.verify = verify
+        self.cert = cert
         self._s = requests.Session()
```

Comment:
Doesn't `requests.Session` contain a field for `cert` already? Then you don't have to pass it in all the GET, POST, PUT, etc. methods

Reply:
Yes, absolutely. But the same happens with the verify attribute. Since the verify attribute was being passed through to each request I decided to do the same thing just to to be consistent. Nevertheless, I agree that it would be better to use Session. 

Should I modify the code and re-submit ?

Thank you for checking this!